src/wallet/rpcwallet.cpp

```diff
@@ -3708,6 +3708,8 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
         return NullUniValue;
     }
 
+    const std::string example_address = "\"bc1qz0wjhhvtlzqec26ftsrwpkwyevjdm7apwctfvx\"";
```

Comment:
I liked that the address was invalid, so it is harder to accidentally send funds to it

Reply:
I'll change it back *and* add a mention in the developer notes rpc section then :smile:

Reply:
Though I agree with @jnewbery that it's nicer when the rpc examples actually work when copy-pasted. It's a trade-off, to be sure.

Reply:
It *does* work. It is an example for an invalid address ;)

Reply:
Done!

Reply:
> so it is harder to accidentally send funds to it

Huh? This is the help text for `getaddressinfo()`. It doesn't send funds.

This help text originally showed a valid address, then was changed to invalid in 17283 (https://github.com/bitcoin/bitcoin/pull/17283#discussion_r350784833), and has now been changed to valid again and invalid again. I don't think it matters that much, but can we just pick one and stick to it?

Also agree that the developer notes should be updated to stop bikeshedding around this.